src/app/left-sidebar-menu/left-sidebar-menu.component.ts

```diff
@@ -1,13 +1,23 @@
 import { Component, inject } from '@angular/core';
 import { DataService } from '../services/data.service';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-left-sidebar-menu',
   standalone: true,
-  imports: [],
+  imports: [CommonModule],
   templateUrl: './left-sidebar-menu.component.html',
-  styleUrl: './left-sidebar-menu.component.css'
+  styleUrls: ['./left-sidebar-menu.component.css']
 })
 export class LeftSidebarMenuComponent {
-  dataService: DataService = inject(DataService);
-}
+  savedCounter: number = 0;
```

Comment:
You can remove this line since you are using the savedCounter from the `data.service.ts` file